css/estilo.css

```
@@ -0,0 +1,7 @@
+/** @format */
+
+body {
+  background: #000000;
```

Rodrigo vc pode colocar apenas o #000;
```suggestion
  background: #000;
```